### DISCUSSION TITLE
Should we close super old / stale issues?

### DISCUSSION THREAD

**Discussion by lwasser** [hozn/stravalib#354] [Team Posts]
hey @stravalib/stravalib-maintainers ! we have some VERY VERY old issues that are 4-6 years old in some cases. do we maybe want to go through and mention, close old ones and then tell the authors who opened them that they can reopen if they are still finding the same issue given we did a bit API change? what do you think? i saw Jonatan did a big issue clean up and that is awesome. what do you think about closing issues with a commen to the author that are maybe older than 1.5 years? idk? we can pick a time frame .

**Comment by jsamoocha** [comment]
Some of these old ones still seem interesting, e.g., documentation requests about webhook subscriptions. I just closed the ones with a question, an answer, and no more comments.
